android/fastlane/Fastfile.private

```diff
@@ -0,0 +1,87 @@
+platform :android do
+
+  desc "Validate"
+  private_lane :validate do |options|
+    # TODO: If the release isn't production, 
+    # validate if there is a user logged on firebase
+    UI.message "Validating..."
+  end
+
+  desc "Builds the app creating the `.apk` or `.aab` files"
+  private_lane :build_application do |options|
+    environment = options[:environment]
+    is_firebase = options[:firebase]
+    version_name = options[:version_name]
+    build_num = options[:build_number]
+
+    UI.message "Forced to version name #{version_name}" if version_name != nil
+    gradle(task: "bumperVersionName", properties: { versionName: version_name }) if version_name != nil
+
+    UI.message "Forced to build number #{build_num}" if build_num != nil
+    gradle(task: "bumperVersionCode", properties: { versionCode: build_num }) if build_num != nil
+      
+    buid_task = get_build_task(environment: environment, firebase: is_firebase)
+    app_Name = get_environment_info(environment: environment)["APP_NAME"]
+
+    gradle(
+      task: buid_task[:task],
+      flavor: buid_task[:flavor],
+      build_type: "Release",
+      properties: {
+        **(is_firebase ? { appName: app_Name } : {})
+      }
+    )
+
+  end
+
+  desc "Distribute the app to Firebase App Distribution"
+  private_lane :distribute_to_firebase do |options|
+    environment = options[:environment]
+
+    desc "Building .apk file..."
+    build_application(
+      environment: environment, 
+      version_name: options[:version_name],
+      build_number: options[:build_number],
+      firebase: true
+    )
+
+    desc "Distributing application to Firebase App Distribution..."
+    firebase_info = get_firebase_info(environment: environment)
+    firebase_app_distribution(
+      app: firebase_info[:app],
+      groups_file: firebase_info[:groups_file]
+    )
+  end
+
+  desc "Release the app to Google Play Console"
+  private_lane :release_to_play_console do |options|
+    environment = options[:environment]
+
+    desc "Building .aab file..."
+    build_application(
+      environment: environment, 
+      version_name: options[:version_name],
+      build_number: options[:build_number],
```

Comment:
And putting the `aab` here. I think that is more clear to have a boolean to build the app in .aab format than a boolean that indicates if is firebase or not. More clear for the `build_application` action